src/Libraries/CoreNodes/List.cs

```diff
@@ -484,8 +484,8 @@ public static IDictionary SortByKey(IList list, IList keys)
 
             return new Dictionary<object, object>
             {
-                { "sorted list", sortedList },
-                { "sorted keys", sortedKeys }
+                { "sortedList", sortedList },
+                { "sortedKeys", sortedKeys }
```

Comment:
These name changes are unrelated to the fix and are made only for consistency so that the same is reflected in the preview bubble.